### DISCUSSION TITLE
Add cancel token support to SDK

### DISCUSSION THREAD

**Discussion by hellojayjay** [directus/directus#6765] [Feature Requests]
### Describe the Bug

when i use sdk, and use default transport(axios internal).but there is no CancelToken in sdk.axios, so how can i cancel an upload reuqest?

### To Reproduce

post /files, want to cancel this request

### What version of Directus are you using?

9.0.0-rc.83

### What version of Node.js are you using?

12.21.0

### What database are you using?

maybe do not care

### What browser are you using?

maybe do not care

### What operating system are you using?

maybe do not care

### How are you deploying Directus?

maybe do not care

**Comment by benhaynes** [comment]
Hey, thanks for asking. Just so you know, questions should be going into GitHub Discussions, Issues are reserved for bug reports.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
